.github/workflows/staging.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: RIF Marketplace PINNER deploy
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-**'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
The same thing here as in the UI, I don't think we should deploy tags to the staging env

Reply:
As we discussed for UI ..can we also merge this one ? .